webbpsf/optics.py

```diff
@@ -841,7 +841,7 @@ def get_transmission(self, wave):
             if poppy.accel_math._USE_NUMEXPR:
                 import numexpr as ne
 
-                jn1 = scipy.special.j1(sigmar)
+                jn1 = scipy.special.j1(sigmar)  # noqa TODO - this looks like a bug, should jn1 be formatted in the following line?
```

Comment:
Not a bug. The following line uses `numexpr` which is a JIT compiler for fast optimized math. It's given a string input which is the expression to compile, then JITs it into byte code which accesses local variables by name.